### DISCUSSION TITLE
How can I configure Keycloak to send emails using AWS SES with IAM roles, instead of using IAM users, to follow best practices?

### DISCUSSION THREAD

**Discussion by mingduan64** [keycloak/keycloak#32010] [Archive]
I have successfully set up Keycloak to send emails via AWS SES using IAM user credentials. However, I would like to follow best practices and configure Keycloak to use IAM roles instead. In the Keycloak admin console, I couldn't find an option to specify IAM roles for email sending. Can you provide detailed instructions on:

Creating and configuring an IAM role with the necessary SES permissions.
Verifying email addresses or domains in AWS SES.
Configuring Keycloak to use the IAM role for sending emails through AWS SES.

**Comment by prifulnath** [comment]
Yeah, I was also thinking like if this was possible in keycloak. It will be a good thing if we can use role-based permission for SES mail sending.
